src/main/resources/db/changesets/20210301_revised_initial_schema.yaml

```diff
@@ -0,0 +1,246 @@
+                primaryKey: true
+                nullable: false
+          - column:
+              name: display_name
+              type: text
+              remarks: |
+                A user-friendly name for the workspace. There is no guarantee of uniqueness across workspaces.
+                Can be null. Omitted for Rawls workspaces, since Rawls maintains its own name
+          - column:
+              name: description
+              type: text
+              remarks: |
+                Free-form text describing the workspace. Can be null. Omitted for Rawls workspaces, since Rawls
+                maintains its own name.
+          - column:
+              name: spend_profile
+              type: text
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: properties
+              type: jsonb
+              remarks: |
+                Key-value map to allow applications to store their data about the workspace
+          - column:
+              name: workspace_stage
+              type: text
+              remarks: |
+                The workspace_stage column is used to hide/expose different features of
+                WSM for different clients. The valid values as of this writing are
+                RAWLS_WORKSPACE and MC_WORKSPACE
+
+    - createTable:
+        tableName: cloud_context
+        columns:
+          - column:
+              name: workspace_id
+              type: text
+              constraints:
+                nullable: false
+                references: workspace(workspace_id)
+                foreignKeyName: fk_cloud_context_wid
+                deleteCascade: true
+              remarks: |
+                Foreign key will cause cloud_context to be deleted when its associated
+                workspace is deleted.
+          - column:
+              name: cloud_platform
+              type: text
+              constraints:
+                nullable: false
+              remarks: |
+                Enum of the cloud: GCP and AZURE are the current types
+          - column:
+              name: context
+              type: jsonb
+              remarks: |
+                JSON form of the information for the specific cloud context
+                
+    - addPrimaryKey:
+        columnNames: workspace_id, cloud_platform
+        tableName: cloud_context
+        constraintName: pk_cloud_context
+        remarks: |
+          Define the PK separately so that we control the order of the columns
+
+    - createTable:
+        tableName: application
+        remarks: |
+          The application table lists all integrated applications configured in the
+          current Terra deployment.
+        columns:
+          - column:
+              name: application_id
+              type: text
+              constraints:
+                primaryKey: true
+                nullable: false
+          - column:
+              name: name
```

Comment:
name is nullable?

Reply:
I updated it to display_name and added remarks for it and the description.